platform-tests/src/test/java/org/junit/platform/console/options/ThemeTests.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.platform.console.options;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.nio.charset.StandardCharsets;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.engine.TestExecutionResult;
+
+class ThemeTests {
+
+	@Test
+	void givenUtf8ShouldReturnUnicode() {
+		assertEquals(Theme.valueOf(StandardCharsets.UTF_8), Theme.UNICODE);
```

Comment:
I think `Theme.valueOf(...)` and `Theme.UNICODE` should be swapped, because `Theme.UNICODE` is the expected value AFAICT?